### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by Tshobohwa**
## In this milestone, we used the pair programming approach to add the following features:
- Preserve the data relative to `books` in a `book.json` file and data relative to `people` in a `people.json` file when the user exits the app.
- Get the data from `books.json` and `people.json` files to populate their relative array when the user starts the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -10,15 +10,20 @@
...
     user_input = gets.chomp.to_i
 
     if OPTIONS.key?(user_input)
       run = OPTIONS[user_input]
+      if run == :exit
+        puts 'Exit'
+        app.store_data_in_files('data/people.json', app.people)
+        app.store_data_in_files('data/books.json', app.books)
+        break
+      end
```
Comment (julie-ify) [inline main.rb]: - [ ] The rentals that were created should also be saved. Only the books and people's data are saved. Kindly implement that of the rentals also.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Tshobohwa and @AbdusSattar-70,

Good job so far!

## Highlights!
- Good code organization and working preservation ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (AbdusSattar-70) [reply main.rb]: @julie-ify Thanks for your suggestion but without creating the rentals array my program is able to show the rentals activities, eventually will we need to implement it?

**Review by omarsalem7 — APPROVED** [review]
Hi @Tshobohwa, @AbdusSattar-70 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**